### PR TITLE
Core: Copy modules to other locale: console command

### PR DESCRIPTION
**Pull request by jeroendesloovere**
## Type

- Feature

## Resolves the following issues

Fixes https://github.com/forkcms/forkcms/issues/2092 and is an addition to https://github.com/forkcms/forkcms/pull/2422

## Pull request description

This PR adds the *console command* `bin/console forkcms:locale:copy` which copies content from the modules from one locale to another.

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -44,6 +44,12 @@ $application = new Application($kernel);
 $application->add(new \Console\Locale\ImportLocaleCommand());
 $application->add(new \Console\Thumbnails\GenerateThumbnailsCommand());
 if ($kernel->getContainer()->get('fork.settings') instanceof ModulesSettings) {
+    $application->add(
+        new \Console\Locale\CopyLocaleCommand(
```
Comment (WouterSioen) [inline bin/console]: I'm not really sure why we're not using use statements here?

File: src/Backend/Modules/Pages/Resources/config/services.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+services:
+    fork.pages.copy_module_to_other_locale:
+        class: Backend\Modules\Pages\Command\CopyModuleToOtherLocale
+        public: true
+        tags:
+            - { name: fork.copy_module_to_other_locale }
```
Comment (WouterSioen) [inline src/Backend/Modules/Pages/Resources/config/services.yml]: It looks like your editor isn't set up to add an EOF character at the end of every file. This is a best practice. 

This should btw be covered by the editorconfig plugin (if you have it installed. PS: you should have it installed 😉 )

File: src/Console/Locale/CopyLocaleCommand.php [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            'To which locale would you like to copy it?',
+            $this->getToLocale($fromLocale)
+        ));
+    }
+
+    private function copyModules(CommonLocale $fromLocale, CommonLocale $toLocale)
+    {
+        $this->copyModulesToOtherLocaleManager->copy($fromLocale, $toLocale);
+
+        $this->io->success('All modules are copied from "' . $fromLocale . '" to "' . $toLocale . '".');
```
Comment (WouterSioen) [inline src/Console/Locale/CopyLocaleCommand.php]: Do all modules support this copy? I don't think that's the case so I would change the returned message to "All supported Modules ..."

File: src/Console/Locale/CopyLocaleCommand.php [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $formatter;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
```
Comment (WouterSioen) [inline src/Console/Locale/CopyLocaleCommand.php]: I don't like abbreviations in code.

Also: why do we have a formatter and an io which is basically the same object?

File: bin/console (hunk 1 above)
Comment (jeroendesloovere) [reply bin/console]: Fixed

**Comment by WouterSioen** [comment]
@jeroendesloovere I don't think your merge did what it was expected to do 😉

File: src/Backend/Modules/Pages/Resources/config/services.yml (hunk 2 above)
Comment (jeroendesloovere) [reply src/Backend/Modules/Pages/Resources/config/services.yml]: Fixed

File: src/Console/Locale/CopyLocaleCommand.php (hunk 3 above)
Comment (jeroendesloovere) [reply src/Console/Locale/CopyLocaleCommand.php]: Fixed

File: src/Console/Locale/CopyLocaleCommand.php (hunk 4 above)
Comment (jeroendesloovere) [reply src/Console/Locale/CopyLocaleCommand.php]: Indeed, removed this variable entirely since we already are using this class as `private $formatter`

**Comment by jeroendesloovere** [comment]
Anybody willing to review this one? @forkcms/core-contributors

**Comment by jeroendesloovere** [comment]
Ping

**Comment by StijnVrolijk** [comment]
Pong.

On a serious note, I'm really busy.

**Comment by carakas** [comment]
is included in #2991

**Pull request closed without merging**